BlackMarket/Model/Product.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Product.swift
+//  BlackMarket
+//
+//  Created by Santiago Andracchi on 20/09/2023.
+//
+
+import Foundation
+import SwiftUI
+
+struct Product: Hashable, Codable, Identifiable {
+  var id: Int
+  var product_picture: String
+  var is_favorite: Bool
+  var unit_price: String
```

Comment:
Snake case style is not suitable for Swift. Please refer to the Swift style guides we promote at Rootstrap for the best practices using the language:
https://rootstrap.github.io/swift/

Reply:
If you need to parse this attributes from a JSON containing snake case keys you can use the CodingKey protocol with Codable: https://developer.apple.com/documentation/swift/codingkey